.travis.yml

```diff
@@ -3,12 +3,10 @@ install: pip install tox
 env:
   - TOX_ENV=docs
   - TOX_ENV=pep8
-  - TOX_ENV=py26-venv-1.7.2
-  - TOX_ENV=py26-venv-1.8.4
-  - TOX_ENV=py26-venv-1.9.1
-  - TOX_ENV=py27-venv-1.7.2
-  - TOX_ENV=py27-venv-1.8.4
-  - TOX_ENV=py27-venv-1.9.1
```

Comment:
hmm is it worth keeping the older versions around?  I get this is technically a version bump, but your original comment does say this won't break existing users' usage.